server/controllers/pokemons/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Pokemon, PokemonClient } from 'pokenode-ts';
+import { APIPokemon, APIPokemonType, COLORS } from '../../models/pokemon/index.js';
+import { APIPageResponse } from '../../models/response/index.js';
+
+const api = new PokemonClient();
+
+function pad(num: number) {
+    if (num > 99) {
+        return `#${num}`
+    }
+
+    if (num > 9) {
+        return `#0${num}`
+    }
+
+    return `#00${num}`
+}
+
+function mapToAPIPokemonType(type: string): APIPokemonType {
+
+    const apiType = Object.keys(APIPokemonType).find((s) => s.toLowerCase() === type.toLowerCase());    
+
+    if (!apiType) {
+        return APIPokemonType.Unknown;
+    }
+    
+    return apiType as APIPokemonType;
+}
+
+function pokeAPIPokemonToAPIPokemon(pokeApiPokemon: Pokemon): APIPokemon {
```

Comment:
Please find another name for this function, is isn't clear enough

Reply:
changed